README.md

```diff
@@ -1,3 +1,5 @@
 # feature-branch-workflow
 
 A simple, boring repo to demonstrate Git workflows involving feature branches.
+
+Bock Bock
```

Comment:
Always with the Bock Bock, Stevie? Get a new catchphrase!

Reply:
Bock Bock Bock!